django/db/models/expressions.py

```diff
@@ -765,6 +782,30 @@ def get_group_by_cols(self):
         return [self]
 
 
+class ExpressionList(Func):
+    """
+    An expression containing multiple expressions. Can be used to provide a
+    list of expressions as an argument to another expression, like an
+    ordering clause.
+    """
+    template = '%(expressions)s'
+    arg_joiner = ', '
+
+    def __init__(self, *expressions, **extra):
+        if len(expressions) == 0:
+            raise ValueError('%s requires at least one expression' % self.__class__.__name__)
+        super().__init__(*expressions, **extra)
+
+    def __repr__(self):
+        extra = ', '.join(str(key) + '=' + str(val) for key, val in self.extra.items())
+        if extra:
+            return '<{}: {}, {}>'.format(self.__class__.__name__, self.__str__(), extra)
+        return '<{}: {}>'.format(self.__class__.__name__, self.__str__())
```

Comment:
`ExpressionList.__repr__()` can be removed because `Func.__repr__()` works in the same way.